src/Integration/TeamExplorer/SectionController.cs

```diff
@@ -259,6 +260,12 @@ private void InitializeProvidedCommands()
             this.ToggleShowAllProjectsCommand = new RelayCommand<ServerViewModel>(this.ToggleShowAllProjects, this.CanToggleShowAllProjects);
             this.BrowseToUrlCommand = new RelayCommand<string>(this.ExecBrowseToUrl, this.CanExecBrowseToUrl);
             this.BrowseToProjectDashboardCommand = new RelayCommand<ProjectViewModel>(this.ExecBrowseToProjectDashboard, this.CanExecBrowseToProjectDashboard);
+            this.ReconnectCommand = new RelayCommand(() =>
```

Comment:
Should this new command also be referenced in the _CleanProvidedCommands_ method below?

Reply:
I don't think it is required. The `ConnectCommand, RefreshCommand and BindCommand` are all coming from different controllers (classes) and I suppose the references are cleaned to prevent memory leaks. For example the DisconnectCommand is not cleaned there...

Reply:
Perhaps the whole `Clean` method is not needed... The class instances that contain the commands are created internally and there is nothing that would keep references... I don't know, it is difficult to see what are the possible implications.